posts/postRouter.js

```diff
@@ -1,27 +1,87 @@
-const express = 'express';
+const express = require("express");
 
-const router = express.Router();
+const Posts = require("./postDb.js");
 
-router.get('/', (req, res) => {
+const router = express.Router();
 
+router.get("/", async (req, res) => {
+  try {
+    const posts = await Posts.get();
+    res.status(200).json(posts);
+  } catch (error) {
+    console.log(error);
+    res.status(500).json({
+      message: "unable to retrive post"
+    });
+  }
 });
 
-router.get('/:id', (req, res) => {
+router.get("/:id", async (req, res) => {
+  try {
+    const posts = await Posts.getById(req.params.id);
 
+    if (posts) {
+      res.status(200).json(posts);
+    } else {
+      res.status(404).json({ message: "post not found" });
+    }
+  } catch (error) {
+    console.log(error);
+    res.status(500).json({
+      message: "unable to retrive post"
+    });
+  }
 });
 
-router.delete('/:id', (req, res) => {
-
+router.delete("/:id", async (req, res) => {
+  try {
+    const posts = await Posts.remove(req.params.id);
+    res.status(200).json(posts);
```

Comment:
You could add a conditional here, if nothing is returned by .remove, then the post doesn't exist. Usually del methods return the amount of rows changed in the database and can be used as a boolean. So instead of `posts` I would call it something like `wasDeleted`.